libsolidity/interface/StandardCompiler.cpp

```diff
@@ -908,6 +908,16 @@ std::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompiler:
 		ret.modelCheckerSettings.engine = *engine;
 	}
 
+	if (modelCheckerSettings.isMember("targets"))
+	{
+		if (!modelCheckerSettings["targets"].isString())
+			return formatFatalError("JSONError", "settings.modelChecker.targets must be a string.");
+		std::optional<ModelCheckerTargets> targets = ModelCheckerTargets::fromString(modelCheckerSettings["targets"].asString());
+		if (!targets)
+			return formatFatalError("JSONError", "Invalid model checker targets requested.");
```

Comment:
Ah oki.. hmm, maybe the error message could be improved so it could also state what targets where not found. this may speedup the finding of a typo.

Reply:
Ok, I think it a more explicit error message is not needed here. It's just the JSON.